web_src/js/components/RepoActionView.vue

```diff
@@ -276,14 +280,16 @@ const sfc = {
       this.fetchPost(`${this.run.link}/approve`);
     },
 
-    createLogLine(line, startTime) {
+    createLogLine(line, startTime, stepIndex) {
       const div = document.createElement('div');
       div.classList.add('job-log-line');
+      div.setAttribute('id', `step-${stepIndex}-${line.index}`);
```

Comment:
Could it have a longer name: eg: `jobstep`

Reply:
Done in https://github.com/go-gitea/gitea/pull/25532/commits/77c550a97b19b54b3d500c6af1f80faa4227f53b